lua/autorun/webaudio.lua

```diff
@@ -36,6 +36,7 @@ end
 -- SERVER
 local WAAdminOnly = CreateConVar("wa_admin_only", "0", FCVAR_REPLICATED, "Whether creation of WebAudio objects should be limited to admins. 0 for everyone, 1 for admins, 2 for superadmins. wa_enable_sv takes precedence over this", 0, 2)
 local WAMaxStreamsPerUser = CreateConVar("wa_stream_max", "5", FCVAR_REPLICATED, "Max number of streams a player can have at once.", 1)
+CreateConVar("wa_autoload_custom_wl", "0", FCVAR_REPLICATED + FCVAR_ARCHIVE, "Auto loads the Custom Whitelist.", 0, 1)
```

Comment:
This should be an object and passed to WebAudio.Common so we access it that way instead of using GetConVar later.

ALTHOUGH -
We shouldn't need a convar to auto load the custom whitelist anyway, I had no idea that it didn't load it. Just slap a loadWhitelist() near where wa_reload_whitelist is defined.

Reply:
Removed the var, and made it just automatically load the custom whitelist